nemo/core/classes/modelPT.py

```diff
@@ -38,7 +38,7 @@
 
     HAVE_MEGATRON_CORE = False
 
-from lightning.pytorch import LightningModule, Trainer
+from pytorch_lightning import LightningModule, Trainer
```

Comment:
revert this change.

Reply:
Why? I am facing issues without this change.

Reply:
it worked well for my runs using the latest container, and I believe it worked for all other domains. We moved to PTL 2+, so better to use `lightning.pytorch`. 
ref: https://www.perplexity.ai/search/lightning-pytorch-trainer-vs-p-cTDOIDUdQhGnuYVtIDudRg?0=d

Reply:
Got it, I reverted this change. Now works fine with the trainer imports you used in the examples/magpietts.py.